src/main/java/com/iridium/iridiumcolorapi/IridiumColorAPI.java

```diff
@@ -34,7 +35,8 @@
      */
     private static final boolean SUPPORTS_RGB = VERSION >= 16;
 
-    private static final List<String> SPECIAL_COLORS = Arrays.asList("&l", "&n", "&o", "&k", "&m","§l", "§n", "§o", "§k", "§m");
+    private static final List<String> SPECIAL_COLORS = Arrays.asList("&l", "&n", "&o", "&k", "&m", "§l", "§n", "§o",
```

Comment:
why'd you add the new line?

Reply:
IDE autoformatting, sorry, I fixed it in the new commit